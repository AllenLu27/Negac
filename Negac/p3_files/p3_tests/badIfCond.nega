void fn(){
	if (1){ }
	if (a){ }
}
