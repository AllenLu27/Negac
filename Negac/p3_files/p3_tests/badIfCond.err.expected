FATAL [3,6]: Undeclared identifier
FATAL [2,6]: Non-bool expression used as an if condition
Semantic analysis failed
